### PR TITLE
clh: update CLH to stable/v0.5.x

### PR DESCRIPTION
**Pull request by likebreath**
Use CLH branch stable/v0.5.x, and also re-generate the openAPI client
code with the new 'cloud-hypervisor.yaml'.

Fixes: #2488

Signed-off-by: Bo Chen <chen.bo@intel.com>

### CODE REVIEW

File: virtcontainers/clh.go [hunk 1]
```diff
@@ -120,6 +120,7 @@ var clhKernelParams = []Param{
 	{"no_timer_check", ""}, // do not check broken timer IRQ resources
 	{"noreplace-smp", ""},  // do not replace SMP instructions
 	{"agent.log_vport", fmt.Sprintf("%d", vSockLogsPort)}, // tell the agent where to send the logs
+	{"rootflags", "data=ordered,errors=remount -ro ro"},   // mount the root filesystem as readonly
```
Comment (sameo) [inline virtcontainers/clh.go]: typo: `remount -ro` should be `remount-ro`.

Comment (likebreath) [reply virtcontainers/clh.go]: @sameo Thanks for correcting me. Just being updated.

Comment (likebreath) [reply virtcontainers/clh.go]: @sameo Just separate the two commits. Any other comments are appreciated. Thanks.

File: virtcontainers/pkg/cloud-hypervisor/client/docs/DiskConfig.md [hunk 2]
```diff
@@ -5,9 +5,13 @@
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
 **Path** | **string** |  | 
+**Readonly** | **bool** |  | [optional] [default to true]
```
Comment (sboeuf) [inline virtcontainers/pkg/cloud-hypervisor/client/docs/DiskConfig.md]: I thought we agreed this shouldn't default to true.

Comment (likebreath) [reply virtcontainers/pkg/cloud-hypervisor/client/docs/DiskConfig.md]: @sboeuf I believe the `default to true` comes from the `cloud-hypervsior.yaml` [here](https://github.com/sameo/cloud-hypervisor-1/blob/76dc26b0d1715b254680cd715273ab682ef437ed/vmm/src/api/openapi/cloud-hypervisor.yaml#L279). Note that all the code under `client/` here are auto-generated based on the yaml file.

Comment (sboeuf) [reply virtcontainers/pkg/cloud-hypervisor/client/docs/DiskConfig.md]: Yes this needs to be fixed from CH directly.

**Comment by amshinde** [comment]
@likebreath Travis is failing with `ERROR: No "Fixes" found`. Can you open an issue and address it with `Fixes :` in the commit message.

**Comment by likebreath** [comment]
> @likebreath Travis is failing with `ERROR: No "Fixes" found`. Can you open an issue and address it with `Fixes :` in the commit message.

@amshinde Thanks for pointing this out. Yes, I am looking at the same failure. I am wondering do I need to put `Fixes xxx` in both commits or just the first commit? Thanks.

**Comment by grahamwhaley** [comment]
You need a Fixes in at least one commit. You can have multiple different ones if you need across multiple commits (but generally we go with one-PR fixes one-Issue, generally)

File: virtcontainers/pkg/cloud-hypervisor/client/docs/DiskConfig.md (hunk 2 above)
Comment (sameo) [reply virtcontainers/pkg/cloud-hypervisor/client/docs/DiskConfig.md]: My bad, this is fixed now. @likebreath You probaly want to fix accordingly.

**Pull request merged**